app/models/read_ribbon.rb

```diff
@@ -0,0 +1,12 @@
+class ReadRibbon < ApplicationRecord
+  belongs_to :user
+  belongs_to :story
+
+  class << self
```

Comment:
Codebase style is to do `def self.hide_replies_for` rather than the nested class definition and I'd like to stick to it.